daemon/src/projection.rs

```diff
@@ -191,6 +191,10 @@ pub struct Cfd {
     #[serde(with = "round_to_two_dp::opt")]
     pub pending_settlement_proposal_price: Option<Price>,
 
+    /// Timestamp when the next fee will be collected
+    #[serde(with = "::time::serde::timestamp::option")]
+    pub next_funding_event: Option<OffsetDateTime>,
```

Comment:
Why do we have to do this in the projection? I think internally we can do that if necessary, but for the projection it would be good to keep it in the `CfdOrder` as it was.
We are displaying it globally in the frontend, and that what it should be.

Reply:
It can't be in the `CfdOrder` as you don't have any timestamp there, so you can't reason about it - or am I missing something?

Reply:
Maybe having it in the `CfdOrder` is confusing, it was a workaround - in the end I think it should always just be the next full hour timestamp. We could also just compute that in the UI for now if that makes it easier.

Reply:
note that it should have the same value for all open cfds. I don't mind it being globally displayed, but then it shouldn't really be inside either Order or Cfd.

maybe it's a non-cfd event?